docs/contributing/contributing-recipes.md

```diff
@@ -0,0 +1,112 @@
+## Contributing to Radius Recipes
+
+This section describes the guidelines for contributing code / docs to Radius Recipes. Contribution can be in the form of adding new recipes for the supported resource types and IaC languages, improving existing recipes or the documentation.
+
+### 1. How to write your first recipe
+
+To write your first recipe, follow the steps below:
+
+1. Familiarize yourself with the IaC language of your choice [Bicep](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep) or [Terraform](https://developer.hashicorp.com/terraform)
+1. Familiarize yourself with the Radius [Recipe](https://docs.radapp.io/recipes) concept
+1. Familiarize yourself with the [supported resource types](https://docs.radapp.io/guides/recipes/overview/#supported-resources) to write recipes for.
+1. Review the existing recipes in this repository to understand the structure and best practices
+1. Follow this [how-to guide](https://docs.radapp.io/guides/recipes/howto-author-recipes/) to write your first recipe, register your recipe in the environment.
+
+### 2. How to test the recipe locally
+
+>[!Note]
+>Since we do not have an automated testing framework for recipes and is in our [backlog](https://github.com/radius-project/recipes/issues/62), we recommend testing the recipe locally in your environment before contributing it to the repository. 
+
+To test the recipe locally, follow the steps below:
+
+1. Publish the recipe to a Registry
+    Recipes leverage [Bicep registries](https://learn.microsoft.com/azure/azure-resource-manager/bicep/private-module-registry) for template storage. Once you've authored a Recipe, you can publish it to your preferred OCI-compliant registry with [`rad bicep publish`]({{< ref rad_bicep_publish >}}):
```

Comment:
{{< >}} shortlinks won't resolve outside of Hugo. You'll need to directly link to the full URL.

Reply:
yes missed to update after copying this from docs